deploy/loderunner/loderunner.yaml

```diff
@@ -1,45 +1,59 @@
-apiVersion: v1
-kind: Pod
-
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: loderunner
+  namespace: loderunner
   labels:
-    app: loderunner
-
+    app.kubernetes.io/name: loderunner
 spec:
-  containers:
-  - name: app
-
-    image: ghcr.io/retaildevcrews/ngsa-lr:beta
-    imagePullPolicy: Always
-
-    args: 
-    - -l
-    - "1000"
-    - --prometheus
-    - -r
-    - -s
-    - http://ngsa-memory:8080
-    - -f
-    - benchmark.json
-    - --zone
-    - dev
-    - --region
-    - dev
-
+  replicas: 1
+  selector:
+    matchLabels:
+      app: loderunner
+  template:
+    metadata:
+      labels:
+        app: loderunner
+    spec:
+      containers:
+        - name: app
+          image: ghcr.io/retaildevcrews/ngsa-lr:beta
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          resources:
+            limits:
+              memory: "512Mi"
+              cpu: "1000m"
+            requests:
+              memory: "512Mi"
+              cpu: "500m"
+          args:
+          - -l
+          - "1000"
+          - -r
+          - -s
+          - http://ngsa-memory.ngsa:8080
+          - http://ngsa-cosmos.ngsa:8080
+          - http://ngsa-java.ngsa:8080
```

Comment:
I think we should only have it configured for ngsa-memory and not the other two, because I wouldn't bother with deploying ngsa-cosmos or ngsa-java for testing in LodeRunner in the local dev cluster.